src/main/java/ir/sahab/elasticsearchrule/ElasticsearchExtension.java

```diff
@@ -0,0 +1,51 @@
+package ir.sahab.elasticsearchrule;
+
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.*;
+
+/**
+ * A JUnit 5 Extension for starting an elasticsearch server instance on the local machine.
```

Comment:
Also explain that the default life-cycle is PER_CLASS and how to change the default behavior